ceph.spec.in

```diff
@@ -478,12 +478,10 @@ Recommends:	python%{_python_buildid}-influxdb
 %endif
 %if 0%{?rhel} == 7
 Requires:       pyOpenSSL
+Requires:       python-enum34
```

Comment:
Instead of having two conditionals `%if 0%{?rhel} == 7` and `%if 0%{?rhel} < 8` it seemed reasonable to have just one, since AFAIK nobody is building nautilus for RHEL 6. Or are they?